### PR TITLE
Update aug release

### PR DESCRIPTION
**Pull request by 0xIslamTaha**
update last release to match the new code base.

### CODE REVIEW

File: api_testing/apis/orders_api.py [hunk 1]
```diff
@@ -108,8 +109,15 @@ def create_new_order(self, **kwargs):
         return api, payload
 
     @api_factory('get')
-    def get_auto_generated_order_no(self):
-        api = '{}{}'.format(self.url, self.END_POINTS['orders_api']['get_auto_generated_number'])
+    def get_auto_generated_order_no(self, year_option="1"):
+        """
+        year_option = 1 : order no with year after (1668-2020)
+        year_option = 0 : order no with year after (2020-1668)
```
Comment (ghada-37) [inline api_testing/apis/orders_api.py:115]: you mean year before

File: api_testing/apis/roles_api.py [hunk 2]
```diff
@@ -85,3 +85,9 @@ def get_random_role(self):
         roles = response['roles']
         editable_roles = [role for role in roles if role['name'] not in ['Admin', 'Contact']]
         return editable_roles
+
+    def get_role_id_from_name(self, name):
+        response, payload = RolesAPI().get_all_roles()
```
Comment (ghada-37) [inline api_testing/apis/roles_api.py:90]: it should be 
`        response, payload = self.get_all_roles()
`

File: api_testing/apis/test_plan_api.py [hunk 3]
```diff
@@ -301,21 +300,22 @@ def create_testplan_from_test_unit_id(self, test_unit_id):
...
         formatted_article = ArticleAPI().get_formatted_article_with_formatted_material_type(formatted_material)
         testplan, payload = self.create_testplan(selectedArticles=[formatted_article],
                                                  materialType=[formatted_material])
-        if testplan['status'] == 1:
-            return (self.get_testplan_form_data(id=testplan['testPlanDetails']['id']))
+        if testplan['message'] == 'operation_success':
+            return (self.get_testplan_form_data(id=payload['number']))
         else:
-            self.info(testplan)
+            raise Exception(f'cant create the test plan with payload {payload}')
```
Comment (ghada-37) [inline api_testing/apis/test_plan_api.py:318]: raise Exception('cant create the test plan with payload {payload}')

File: ui_testing/pages/base_pages.py [hunk 4]
```diff
@@ -408,6 +408,8 @@ def set_all_configure_table_columns_to_specific_value(self, value=True, always_h
 
     def deselect_all_configurations(self):
         self.open_configure_table()
+        import ipdb;
+        ipdb.set_trace()
```
Comment (ghada-37) [inline ui_testing/pages/base_pages.py:412]: remove ipdb before merge

File: api_testing/apis/orders_api.py (hunk 1 above)
Comment (0xIslamTaha) [reply api_testing/apis/orders_api.py:115]: done

File: api_testing/apis/roles_api.py (hunk 2 above)
Comment (0xIslamTaha) [reply api_testing/apis/roles_api.py:90]: done

File: api_testing/apis/test_plan_api.py (hunk 3 above)
Comment (0xIslamTaha) [reply api_testing/apis/test_plan_api.py:318]: what do u mean?

File: ui_testing/pages/base_pages.py (hunk 4 above)
Comment (0xIslamTaha) [reply ui_testing/pages/base_pages.py:412]: ops :D

File: api_testing/apis/test_plan_api.py (hunk 3 above)
Comment (ghada-37) [reply api_testing/apis/test_plan_api.py:318]: what is "f" letter before "cant..."?

**Pull request merged**